tests/fixtures/decam.py

```diff
@@ -440,8 +440,8 @@ def decam_reference(decam_ref_datastore):
         ref = Reference()
         ref.image = ds.image
         ref.provenance = prov
-        ref.validity_start = Time(50000, format='mjd', scale='utc').isot
-        ref.validity_end = Time(65000, format='mjd', scale='utc').isot
+        ref.validity_start = Time(55000, format='mjd', scale='utc').isot
+        ref.validity_end = Time(62000, format='mjd', scale='utc').isot
```

Comment:
If I set the time to 65000 I get an ERFA warning.

Reply:
Use scale='tai' instead of scale='utc', and that warning will go away.

Reply:
(This is a mismatch betwen UTC and TAI ; I think ISO times are defined to be TAI.  The difference is currently somethign like half a minute.  It will go up in time, but for what we have right here, we don't care at that level.)